Makefile

```diff
@@ -86,7 +86,7 @@ $(LDSO_PATHNAME): $(LOBJS)
 		-shared -o $@ $(LOBJS) $(SOCKET_LIBS)
 
 $(ALL_TOOLS): $(OBJS)
-	$(CC) src/main.o src/common.o $(USER_LDFLAGS) -o $(PXCHAINS)
+	$(CC) src/main.o src/common.o $(USER_LDFLAGS) $(LIBDL) -o $(PXCHAINS)
```

Comment:
where does LIBDL come from ?

Reply:
The default value of LIBDL is defined at https://github.com/rofl0r/proxychains-ng/blob/master/Makefile#L27. On FreeBSD, the value is cleared at https://github.com/rofl0r/proxychains-ng/blob/master/configure#L210

Reply:
oops, missed that :)